index.bs

```diff
@@ -98,8 +98,13 @@ Do explain the connection,
 even if this requires breaking the “be brief” rule.
 For example, see the
 [explainer for deprecating `document.domain`](https://github.com/mikewest/deprecating-document-domain/#a-problem),
-although even that could perhaps use another sentence
-explaining why security boundaries are important for users.
+and see [[#boilerplate-user-benefit]] for text to
+explain why security boundaries are important for users.
+
+Some features are intended to help authors and
+have no effect on end-users aside from websites being developed more easily.
+If your feature doesn't cause any harm or benefit to users,
+use [[#authoring-user-benefit|the boilerplate for ease of authoring]].
```

Comment:
Did you mean to reference the subsection of the appendix or the entire thing?  It seems like this should be the entire appendix.

Reply:
I meant to reference the subsection, since I meant this paragraph to just be about features that are meant to make authoring easier. The other parts of the appendix explain why features meant for security, etc, actually do help users. But I'm also not tied to the current text or link if you still prefer something else.

Reply:
@lolaodelola  endorsed the subsection link, so I'll merge. We can fix in another PR if necessary.